Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -27,7 +26,6 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
 
     QTextStream(stdout) << "Voltage: " << batteryData.voltage
         << " Current: " << batteryData.current
-        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed() << endl;
-
-    // TODO: Print out time until the battery is fully charged or depleted.
+        << " Total Ah used: " << batteryStateOfChargeService_.totalAmpHoursUsed()
+        << "TIME UNTIL BATTERY FULLY CHARGED" << batteryStateOfChargeService_.timeWhenChargedOrDepleted().toString()<<endl;
```

Comment:
Here is a sample output:
`Voltage: 3.11293 Current: 0.530217 Total Ah used: 5.47108e+249TIME UNTIL BATTERY FULLY CHARGED02:12:02`
There are 2 errors here, can you figure out what they are?